docs/docs/aleph_zero/sdk.md

```diff
@@ -47,25 +33,41 @@ SDK also contains all essential values and helper functions that might be needed
  ┗ 📂tests
 ```
 
-### Contracts
+### Transactions and Queries
+
+When working with contracts, developers can seamlessly initiate interactions by calling methods from the corresponding contract class. The first parameter designates the account, and subsequent parameters act as entrypoint parameters, facilitating smooth contract interactions.
```

Comment:
But here, there is lack of description of what is in the sub-section name (Transactions and Queries), which involves explaining the occurrence of two types of **transaction** calls and **queries**